recipes/itk-dev/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "4.11.0" %}
+
+package:
+  name: itk-dev
+  version: "{{version}}"
+
+source:
+  url: https://github.com/InsightSoftwareConsortium/ITK/archive/v{{version}}.tar.gz
```

Comment:
As the linter says, we need a checksum. Preferably `sha256`. Preferably not `md5`. Though others could work too.